packages/passport/sdk-sample-app/src/components/zkevm/EthSendTransactionExamples/SeaportFulfillAvailableAdvancedOrders.tsx

```diff
@@ -0,0 +1,172 @@
+import { PopulatedTransaction } from 'ethers';
+import React, {
+  useCallback, useEffect, useMemo, useState,
+} from 'react';
+import {
+  Accordion, Form, Spinner, Stack,
+} from 'react-bootstrap';
+import WorkflowButton from '@/components/WorkflowButton';
+import { RequestExampleProps } from '@/types';
+import { useImmutableProvider } from '@/context/ImmutableProvider';
+import { usePassportProvider } from '@/context/PassportProvider';
+import { ActionType, TransactionAction, TransactionPurpose } from '@imtbl/orderbook';
+
+function SeaportFulfillAvailableAdvancedOrders({ disabled, handleExampleSubmitted }: RequestExampleProps) {
+  const { orderbookClient } = useImmutableProvider();
+  const { zkEvmProvider } = usePassportProvider();
+
+  const [listingIds, setListingIds] = useState<string>('');
+  const [walletAddress, setWalletAddress] = useState<string>('');
+  const [isBuldingTransaction, setIsBuildingTransaction] = useState<boolean>(false);
+  const [transaction, setTransaction] = useState<PopulatedTransaction>();
+  const [transactionError, setTransactionError] = useState<string>('');
+
+  const seaportContractAddress = useMemo(
+    () => (
+      orderbookClient.config().seaportContractAddress
+    ),
+    [orderbookClient],
+  );
+
+  useEffect(() => {
+    const getAddress = async () => {
+      if (zkEvmProvider) {
+        const [address] = await zkEvmProvider.request({
+          method: 'eth_requestAccounts',
+        });
+        setWalletAddress(address || '');
+      }
+    };
+
+    getAddress().catch(console.log);
+  }, [zkEvmProvider, setWalletAddress]);
+
+  useEffect(() => {
+    setTransactionError('');
+    setTransaction(undefined);
+  }, [listingIds]);
+
+  const validate = useCallback(async () => {
+    setTransactionError('');
+    setTransaction(undefined);
+    setIsBuildingTransaction(true);
+
+    try {
+      const fulfillResponse = await orderbookClient.fulfillBulkOrders(
+        listingIds.replace(' ', '').split(',').map((orderId) => ({
+          listingId: orderId,
+          takerFees: [],
+        })),
+        walletAddress,
+      );
+
+      if (!fulfillResponse.sufficientBalance) {
+        setTransactionError('Insufficient balance to fulfill the order(s)');
+        return;
+      }
+
+      if (fulfillResponse.sufficientBalance) {
```

Comment:
```suggestion
```
can we remove it?

Reply:
Oh good catch 😅 must have been a refactor gone wrong. Fixed ✅